Code/Test_Model_On_Kaggle.py

```diff
@@ -0,0 +1,15 @@
+from torch.utils.data import DataLoader
+
+from Code.KaggleDataset import KaggleDataset
+from Code.testing_common_utils import predict_on_test_data, model_uzeros
+from Code.training_common_utils import image_preprocessing
+
+kaggle_data = KaggleDataset(
+    csv_file="../Data/sample/sample_labels.csv",
+    root_dir="../Data/sample/images",
+    image_transform=image_preprocessing
+)
+
+kaggle_data_loader = DataLoader(kaggle_data)

```

Comment:
For this and the one above. 
Do we not want to specify a batch size?

Reply:
I think that only matters when training.  I don't see what it would mean when predicting on test data